pkg/appmanager/as3Manager.go

```diff
@@ -52,6 +63,42 @@ func (appMgr *Manager) processUserDefinedAS3(template string) bool {
 	return true
 }
 
+// Validates the AS3 Template
+func (appMgr *Manager) validateAS3Template(template string) bool {
+
+	workingDir, _ := os.Getwd()
+	var schemaUrl = workingDir + "/vendor/src/f5/schemas/as3-schema-3.9.0-3-cis.json"
```

Comment:
Suggested use of `appMgr.schemaLocal`

https://github.com/F5Networks/k8s-bigip-ctlr/blob/f1e115ed7a1ccce2767df9b6afc9fe28d9ffb80b/pkg/appmanager/resourceConfig.go#L1285